packages/dart/sshnoports/bin/sshnp.dart

```diff
@@ -168,7 +168,7 @@ void main(List<String> args) async {
       // Run List Devices Operation
       if (params.listDevices) {
         stderr.writeln('Searching for devices...');
-        var deviceList = await sshnp.listDevices();
+        SshnpDeviceList deviceList = await sshnp.listDevices();
```

Comment:
Made this type explicit, because I wrote this code, and I was sure it was a custom object, but the name I chose suggested otherwise... oops...